script/deploy/safe/confirm-safe-tx.ts

```diff
@@ -1,173 +1,61 @@
+/**
```

Comment:
when I goes through all networks at the end I get an error. Why is that?

![image](https://github.com/user-attachments/assets/501e2371-4d4d-40e0-9951-7240deafc017)

Reply:
Probably just an RPC error. Worked fine for me.

Reply:
ok!